README.md

```diff
@@ -60,21 +106,70 @@ Parameters:
 
 Returns a `FirebaseDeviceStore`.
 
-#### `FirebaseDeviceStore.signOut(): void`
+#### `signOut(completion: (Error?) -> Void): void`
 
 Indicate to the DeviceStore that the user is about to sign out, and the current device token should be removed.
 
-This cannot be done automatically with `onAuthStateChanged` as the user won't have permission to remove the token from Firestore as they are already signed out by this point and the Cloud Firestore security rules will prevent the database deletion.
+This can't be done automatically with `onAuthStateChanged` as the user is already signed out at this point. This means the Cloud Firestore security rules will prevent the database deletion as they no longer have the correct user permissions to remove the token.
+
+Parameters:
+
+- `completion`: `(Error?) -> Void` a callback handler which will return an `Error` if signing out wasn't successful, or `nil` otherwise
```

Comment:
Perhaps "will return" should be "will be called with" ?  Or do iOS completion handlers regularly phrase it as "return"?

Reply:
Updated